### DISCUSSION TITLE
Fastlane match to clone project gitlab CI_JOB_TOKEN

### DISCUSSION THREAD

**Discussion by ziadtamim** [fastlane/fastlane#20704] [Q&A]
Hello All, 

Performing Fastlane match for deployment requires access to the repository where certificates and provisioning profiles are stored. The issue is that the lane that runs the Fastlane match triggered by a Gitlab Runner only has access to the repository it was triggered from and not the one where the certificates are stored.

How to use Fastlane match to customize repository clone using `CI_JOB_TOKEN` environment variable?

Thanks in advance.

**Comment by macrozone** [comment]
@ziadtamim did you find a solution? i would also like to know

**Comment by macrozone** [comment]
ok i found the solution

in Fastfile do this:

```
CI_JOB_TOKEN = ENV["CI_JOB_TOKEN"]
git_url("https://gitlab-ci-token:"+CI_JOB_TOKEN+"@your-gitlab-domain.com/path/to/certificate-repo")
```
